.node-version

```diff
@@ -1 +1 @@
-16.20.0
+18.15.0
```

Comment:
Can you also bump this in the package.json?

Reply:
In which package.json?

I did in root

```
"engines": {
    "npm": ">=9.0.0 <10.0.0",
    "node": ">=18.0.0 <19.0.0"
  }
```

Reply:
Missed that. My bad!